### PR TITLE
Changelog aug 31

### PR DESCRIPTION
**Pull request by AleksandarCole**

### CODE REVIEW

File: docs/ci-cd-environment/ubuntu-18.04-image.md [hunk 1]
```diff
@@ -66,8 +66,8 @@ Following version control tools are pre-installed:
 
 - Firefox 78.1
 - geckodriver 0.26.0
-- Google Chrome 83
-- Chrome_driver 83
+- Google Chrome 84
+- Chrome_driver 84
```
Comment (AleksandarCole) [inline docs/ci-cd-environment/ubuntu-18.04-image.md:70]: @bogyo210 please confirm this update is ok.

File: docs/reference/semaphore-changelog.md [hunk 2]
```diff
@@ -75,6 +103,7 @@ Added possibility to change firefox version with the `sem-version` tool: `sem-ve
   - PHP 7.2.31 -> 7.2.32
   - PHP 7.3.19 -> 7.3.20
   - PHP 7.4.7 -> 7.4.8
+  - Chrome_driver 83 > 84
```
Comment (AleksandarCole) [inline docs/reference/semaphore-changelog.md:106]: @bogyo210 please confirm this update is ok.

**Review by bogyo210 — APPROVED** [review]

File: docs/reference/semaphore-changelog.md [hunk 3]
```diff
@@ -8,15 +8,35 @@ Thank you for using Semaphore!
 We continuously deploy changes that improve the product for you.
 This page is updated on a weekly basis.
 
+### Week of August 31, 2020
+**(Improved) New Semaphore UI public beta release**  
+After successful private beta the new Semaphore UI has been rolled out to a larger number of users.
```
Comment (Ivana-Markovic) [inline docs/reference/semaphore-changelog.md:13]: => After a successful private beta, the new Semaphore UI has been rolled out to a larger number of users.

File: docs/reference/semaphore-changelog.md [hunk 4]
```diff
@@ -8,15 +8,35 @@ Thank you for using Semaphore!
 We continuously deploy changes that improve the product for you.
 This page is updated on a weekly basis.
 
+### Week of August 31, 2020
+**(Improved) New Semaphore UI public beta release**  
+After successful private beta the new Semaphore UI has been rolled out to a larger number of users.  
+The new UI brings several improvements:  
+
+- Easier navigation through better breadcrumbs and new "Jump to" element.
```
Comment (Ivana-Markovic) [inline docs/reference/semaphore-changelog.md:16]: => Easier navigation through better breadcrumbs and a new "Jump to" element.

File: docs/reference/semaphore-changelog.md [hunk 5]
```diff
@@ -8,15 +8,35 @@ Thank you for using Semaphore!
...
+- Additional information in workflow lists allows you to quickly see the status of deployments.
+- Enhanced Workflow builder - new options like job prioritization have been added in Workflow builder.
```
Comment (Ivana-Markovic) [inline docs/reference/semaphore-changelog.md:18]: => Enhanced Workflow builder - new options like job prioritization have been added to the Workflow builder.

File: docs/reference/semaphore-changelog.md [hunk 6]
```diff
@@ -8,15 +8,35 @@ Thank you for using Semaphore!
...
+- Improved job logs with collapsable command lines and dark theme.
+- Activity Monitor page that provides a handy overview of your running pipelines and quota limits.
+
+You can read more about new UI in our [latest blog post](https://semaphoreci.com/blog/refreshed-new-ui-for-a-greater-experience).  
+If you want to try out new UI but it's not yet rolled out to your organization please reach out to [our support team](mailto:support@semaphoreci.com).
```
Comment (Ivana-Markovic) [inline docs/reference/semaphore-changelog.md:23]: => If you want to try out the new UI but it's not yet rolled out to your organization, please reach out to [our support team](mailto:support@semaphoreci.com).

**Pull request merged**